packages/main/src/types/BreadcrumbsDesign.js

```diff
@@ -0,0 +1,42 @@
+import DataType from "@ui5/webcomponents-base/dist/types/DataType.js";
+
+/**
+ * @lends sap.ui.webcomponents.main.types.BreadcrumbsDesign.prototype
+ * @public
+ */
+const BreadcrumbsTypes = {
+	/**
+	 * Shows the current page as the last item in the trail.
+	 * The last item contains only plain text and not a link.
+	 *
+	 * @public
+	 * @type {Standard}
+	 */
+	Standard: "Standard",
+
+	/**
+	 * All items in the breadcrumb are links.
+	 * @public
+	 * @type {NoCurrentPage}
+	 */
+	NoCurrentPage: "NoCurrentPage",
+};
+
+/**
+ * @class
+ * Different types of Breadcrumbs.
+ * @constructor
+ * @author SAP SE
+ * @alias sap.ui.webcomponents.main.types.BreadcrumbsDesign
+ * @public
+ * @enum {string}
+ */
+class BreadcrumbsDesign extends DataType {
```

Comment:
Will double-check with the Core guys if "since" tags need to be specified on fields from the initial implementation
(when browsing the history, I see earlier committers did not specify since tags in those cases, but needs to be communicated for clarity)